source/androidx.recyclerview/recyclerview/transforms/Metadata.xml

```diff
@@ -2,6 +2,10 @@
     <attr path="/api/package[@name='androidx.recyclerview.widget']/class[@name='LayoutState']" name="visibility">public</attr>
     <attr path="/api/package[@name='androidx.recyclerview.widget']/class[@name='RecyclerView.Adapter']/method[@name='bindViewHolder']/parameter[1]" name="type">androidx.recyclerview.widget.RecyclerView.ViewHolder</attr>
     <attr path="/api/package[@name='androidx.recyclerview.widget']/class[@name='RecyclerView.Adapter']/method[@name='onBindViewHolder']/parameter[1]" name="type">androidx.recyclerview.widget.RecyclerView.ViewHolder</attr>
-    <attr path="/api/package[@name='androidx.recyclerview.widget']/class[@name='RecyclerView.Adapter']/method[@name='onCreateViewHolder']" name="return">androidx.recyclerview.widget.RecyclerView.ViewHolder</attr>
     <remove-node path="/api/package[@name='androidx.recyclerview.widget']/interface[@name='RecyclerView.ItemAnimator.ItemAnimatorListener']" />
+    
+    <!--
+    <attr path="/api/package[@name='androidx.recyclerview.widget']/class[@name='RecyclerView.Adapter']/method[@name='onCreateViewHolder']" name="return">androidx.recyclerview.widget.RecyclerView.ViewHolder</attr>
```

Comment:
Same as above, no real value to keeping lines that are not needed. Unless this is just because the PR is a WIP?